test/index.js

```diff
@@ -175,6 +177,17 @@ describe('PubSub', function() {
       fakeUtil.normalizeArguments = normalizeArguments;
     });
 
+    it('should combine all required scopes', function() {
+      v1ClientOverrides.SubscriberClient = util.noop;
+      v1ClientOverrides.SubscriberClient.scopes = ['a', 'b', 'c'];
+
+      v1ClientOverrides.PublisherClient = util.noop;
+      v1ClientOverrides.PublisherClient.scopes = ['a', 'b', 'c', 'd', 'e'];
+
+      var pubsub = new PubSub({});
+      assert.deepEqual(pubsub.options.scopes, ['a', 'b', 'c', 'd', 'e']);
```

Comment:
would be better to add some unique scope to SubscriberClient, otherwise you don't actually check that it's a union of two sets

Reply:
Great point. Would you mind making that change?